src/FriendsPage.tsx

```diff
@@ -24,24 +89,20 @@ const FriendsPage: React.FC = () => {
         }
       )
       .catch((error) => {
-        setIsLoaded(true);
         setError(error);
+      })
+      .finally(() => {
+        setIsLoaded(true);
       });
   }, []);
 
   const friendRequests: Friend[] = useMemo(
-    () =>
-      friends.filter(
-        (friend: Friend) => friend.status === FriendStatus.requested
-      ),
+    () => friends.filter((friend: Friend) => friend.status === "requested"),
     [friends]
   );
 
   const currentFriends: Friend[] = useMemo(
-    () =>
-      friends.filter(
-        (friend: Friend) => friend.status === FriendStatus.accepted
-      ),
+    () => friends.filter((friend: Friend) => friend.status === "friends"),
```

Comment:
Is there a specific reason you think we shouldn't be using enums here?

Reply:
The database can accept strings or number values as inputs (like for PUT or POST requests), but it only returns strings for GET requests. Enums use number comparison (1 === "friends") which won't work here.

I don't think there will be any issues because the Friend interface in ourtypes.tsx only allows for explicitly-set strings and typescript will complain if it's not one of those.

Reply:
Ok, good to know! I'll update my code accordingly since I added in the a book interface in the modal branch